### DISCUSSION TITLE
18GB RAM

### DISCUSSION THREAD

**Discussion by Isanderthul** [guydavis/machinaris#688] [General]
seems like this ram usage is too high
<img width="1476" alt="Screenshot 2022-06-07 at 16 50 17" src="https://user-images.githubusercontent.com/3693197/172411537-82e2be17-96f5-419a-bd44-383e24ab8ec2.png">
<img width="1501" alt="Screenshot 2022-06-07 at 16 56 06" src="https://user-images.githubusercontent.com/3693197/172412556-655246d3-a66b-4e00-ae42-4becf52b7c6d.png">

anyone else getting this?

**Comment by guydavis** [comment]
Hi, that is pretty high yes.  Here's what I see on my tower: `docker stats machinaris machinaris-btcgreen machinaris-cactus machinaris-cryptodoge machinaris-chives machinaris-flax machinaris-flora machinaris-hddcoin machinaris-maize machinaris-mmx machinaris-nchain machinaris-shibgreen machinaris-silicoin machinaris-staicoin machinaris-stor`

![image](https://user-images.githubusercontent.com/12936/172413420-8d350b41-3109-4cfe-bec2-de7b93a748dc.png)

![image](https://user-images.githubusercontent.com/12936/172413623-5d68aae4-455b-4e7e-97c1-e2bd24e0288a.png)

![image](https://user-images.githubusercontent.com/12936/172413787-ada2a5f2-49d8-42b1-8231-73b107e01df9.png)

How long has your Machinaris container been up? Try a restart I think.  Hope this helps.

**Comment by Isanderthul** [comment]
OK uptime is 7 days. Also it is running DB V1. I will restart it, and move across to DB V2
<img width="1889" alt="Screenshot 2022-06-09 at 09 06 21" src="https://user-images.githubusercontent.com/3693197/172785972-db407c6b-e49d-4474-bfc5-e531688234e7.png">
